ckanext/datitrentinoit/helpers.py

```diff
@@ -40,11 +40,32 @@ def recent_updates(n):
         log.info(':::::::::::: Retrieving the corresponding localized title and abstract :::::::::::::::')
 
         lang = multilang_helpers.getLanguage()
-        
-        q_results = model.Session.query(PackageMultilang).filter(PackageMultilang.package_id == item.get('id'), PackageMultilang.lang == lang).all() 
+
+        q_results = model.Session.query(PackageMultilang)\
+                                 .filter(PackageMultilang.package_id == item.get('id'),
+                                         PackageMultilang.lang == lang).all()
 
         if q_results:
             for result in q_results:
                 item[result.field] = result.text
-	
-    return search_results.get('results', [])
\ No newline at end of file
+
+    return search_results.get('results', [])
+
+# this is a hack against ckan-2.4.0 (until 2.4.7)
```

Comment:
Here you reported (until 2.4.7) but in Trento VM we have ckan 2.5.2
This piece of code is needed also in ckan 2.5.2? If yes probably we need to fix the comment.

Reply:
Yes, because 2.5 was broken here too. Fix was introduced in 2.5.6: https://github.com/ckan/ckan/commit/44a82d732e8be97ac2df03c11b2659ff1a4d6e30

Reply:
ok, so please fix this comment. thank you